.env-example

```diff
@@ -11,7 +11,8 @@ GS_PROJECT_ID="gcp-project-name-here"
 GOOGLE_APPLICATION_CREDENTIALS="./local-credentials/add-your-service-account-credentials-here.json"
 
 # SSO / OIDC configuration
-USE_SSO_AUTH=False  # By default for local running, go with username+password auth, not SSO
```

Comment:
This is unrelated, but I couldn't run birdbox locally because it would choke on this line (interpreting everything after `=` as the value of the variable).